strawberryfields/backends/tfbackend/backend.py

```diff
@@ -82,10 +82,13 @@ def begin_circuit(self, num_subsystems, **kwargs):
                 For each mode, the simulator can represent the Fock states :math:`\ket{0}, \ket{1}, \ldots, \ket{\text{cutoff_dim}-1}`.
             pure (bool): If True (default), use a pure state representation (otherwise will use a mixed state representation).
             batch_size (None or int): Size of the batch-axis dimension. If None, no batch-axis will be used.
+            dtype (DType): Complex Tensorflow Tensor type representation, either complex64 (default) or complex128.
+                Note, complex128 will increase memory usage substantially.
```

Comment:
@Aaron-Robertson is this something you noticed during testing?

Reply:
I noticed a small increase, but mostly included because it's noted in the issue(s).